packages/odyssey-storybook/src/components/Modal/Modal.stories.tsx

```diff
@@ -77,7 +77,7 @@ const Template: Story<ModalProps> = () => {
 
 export const Default = Template.bind({});
 Default.args = {
-  open: true,
+  open: false,
```

Comment:
will this affect our visual regression testing runs? As the modal will be closed now rather than open?

Reply:
Ah, this is probably why the Applitools tests are failing in this PR -- will revert this change.